### ISSUE TITLE
Split strings in the password dialog

### ISSUE THREAD

**Issue by piotrdrag** [rhinstaller/anaconda#1258]
#1206 introduced combined/split strings, which are highly problematic to localization. That part of the change needs to be reverted/rewritten, otherwise translators might not be able to translate those strings.

More info on the topic: https://wiki.gnome.org/TranslationProject/DevGuidelines/Never%20split%20sentences

**Comment by M4rtinK** [comment]
Where exactly you see that ? I don't see anything like the Gnome Wiki example in the new [password message strings in constants](https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/constants.py#L139). It's all complete sentences, even with named placeholders.

**Comment by piotrdrag** [comment]
It’s not a complete sentence if the subject is a variable. :)

**Comment by piotrdrag** [comment]
Ping? This is a real problem that blocks any translation efforts.

**Comment by M4rtinK** [comment]
So what's the suggested solution ? Spell the name in full:

```
PASSWORD_EMPTY_ERROR = N_("The password is empty.")  # singular
PASSPHRASE_EMPTY_ERROR = N_("The passphrase is empty.")  # singular
PASSWORD_CONFIRM_ERROR_GUI = N_("The passwords do not match.")  # plural
PASSPHRASE_CONFIRM_ERROR_GUI = N_("The passphrases do not match.")  # plural
```

That roughly douples the number of translatable strings for the password checking constants, but I guess that's not a problem ?

**Comment by piotrdrag** [comment]
Yes, that is right.

And no, it’s not a problem. It’s why we have translation memory and fuzzy marching. :)

**Comment by piotrdrag** [comment]
Fixed by #1282.
